### PR TITLE
feat(content-server): Easter egg for Cloud Services

### LINKED ISSUE #2470: Add a tabzilla-like console log message that makes it easy to find us/recruit contributors
I always loved this, and think it's a great way to help developers find us and recruit contributors.

From www.mozilla.org:

screen shot 2015-05-25 at 18 13 39

In additional to the standard Mozilla template, we could add some FxA specific text.

### PR DESCRIPTION
**Pull request by TDA**
Fixes #2470 
@shane-tomlinson 
@vladikoff

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b85be5cefa8eff0dfe18e50bbb58606d5d619417
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: bower.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "blueimp-canvas-to-blob": "2.1.0",
     "chai": "1.8.1",
     "cocktail": "0.5.9",
+    "easteregg": "https://github.com/mozilla/fxa-easter-egg.git",
```
Comment (vladikoff) [inline bower.json]: Lock to some hash here

**Comment by rfk** [comment]
What's the next action on this bug, is it ready to merge?

**Comment by shane-tomlinson** [comment]
> What's the next action on this bug, is it ready to merge?

Not yet, @TDA and I are going through the easter egg repo with a fine tooth comb and cleaning it up.

**Pull request merged**